src/encoded/static/components/matrix_experiment.js

```diff
@@ -140,6 +144,9 @@ const convertExperimentToDataTable = (context, getRowCategories, getRowSubCatego
         const renderedData = categoryExpanded ? subCategoryData : subCategoryData.slice(0, SUB_CATEGORY_SHORT_SIZE);
         matrixRowKeys[matrixRow] = rowCategoryBucket.key;
         matrixRow += 1;
+        const cellDataUrl = searchBaseUrlComponents
+            .filter(s => !(s.includes('biosample_ontology.term_name') || s.includes('assay_title')))
+            .join('&');
```

Comment:
I think hardcoding these strings is risky in case they change in the page JSON. “biosample_ontology.term_name” is already in the `subCategory` variable, and “assay_title” is already in the `columnCategoryType` variable. The same applies to the first change above for “assay_title” and the changes below for “biosample_ontology.term_name” and “assay_title.”

Reply:
Good catch. I am tend to use magic strings so that thought did cross my mind. I fixed this.